src/index.css

```diff
@@ -1,68 +1,23 @@
-:root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
-  line-height: 1.5;
-  font-weight: 400;
-
-  color-scheme: light dark;
-  color: rgba(255, 255, 255, 0.87);
-  background-color: #242424;
-
-  font-synthesis: none;
-  text-rendering: optimizeLegibility;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+.login {
+    background-image: url('assets/bg.svg');
+    background-size: covecontainr;
+    background-position: center center;
 }
 
-a {
-  font-weight: 500;
-  color: #646cff;
-  text-decoration: inherit;
-}
-a:hover {
-  color: #535bf2;
+.form-bg{background: rgba(49, 89, 81, 0.9);
```

Comment:
We need to avoid having fixed colors and use css variables instead, as we will need to implement light/dark mode feature. Having fixed colors will make this very difficult to implement.